addons/web/static/src/views/fields/boolean_toggle/boolean_toggle_field.js

```diff
@@ -9,12 +9,21 @@ export class BooleanToggleField extends BooleanField {
         return this.props.record.isReadonly(this.props.name);
     }
     onChange(newValue) {
-        this.props.update(newValue, { save: true });
+        this.props.update(newValue, { save: this.props.autosave });
     }
 }
 
 BooleanToggleField.template = "web.BooleanToggleField";
 
 BooleanToggleField.displayName = _lt("Toggle");
+BooleanToggleField.props = {
+    ...BooleanField.props,
+    autosave: { type: Boolean, optional: true },
+};
+BooleanToggleField.extractProps = ({ attrs }) => {
+    return {
+        autosave: "autosave" in attrs ? Boolean(attrs.autosave) : true,
```

Comment:
I think this should be in options, not as attribute. (and you wouldn't have to change the rng if it was in options)